git/README.md

```diff
@@ -108,14 +108,17 @@ Once your PR has been approved, you are responsible for merging it.
 
 ### Rebasing
 
-Squash commits like "Fix whitespace" into one or a small number of valuable
-commit(s). Edit commit messages to reveal intent. Run tests.
+[Squash commits](https://cloudfour.com/thinks/squashing-your-pull-requests/)
+like "Fix whitespace" into one or a small number of valuable commit(s). 
+Edit commit messages to reveal intent. Run tests.
 
 ```
 git fetch origin
 git rebase -i origin/master
 ```
 
+- [How and Why to Squash Merge your Pull Request](https://cloudfour.com/thinks/squashing-your-pull-requests/)
+
```

Comment:
Hey @spaceninja , should this have a separate heading from `Rebasing`? It seems odd to have this structure:

- Rebasing heading
- Squashing info
- Rebasing example code
- Squashing link

Reply:
That's an excellent point. The doc originally mentioned squashing in the context of rebasing your branch before merging, but my blog post is really about the squash merge tool on github. I'll move my link a little lower and clarify this text.

Reply:
Thanks!

Reply:
@Paul-Hebert updated, thanks for the suggestion!